src/styles/viewerToolbar.styl

```diff
@@ -1,71 +1,79 @@
 @require 'cozy-ui'
 
+
 .pho-viewer-toolbar
-    height      em(90px)
-    flex-shrink 0
+    z-index     101
     display     flex
+    flex-shrink 0
     width       100vw
-    z-index     101
+    height      em(90px)
 
     &::before
-        content     ''
-        width       10em
-        font-size   14px
         flex-grow   0
+        width       10em
+        content     ''
+        font-size   em(14px)
 
-    .pho-viewer-toolbar-actions
-        display flex
-        width   100%
 
+.pho-viewer-toolbar-actions
+    display flex
+    width   100%
+
+
+.pho-viewer-toolbar-close
+    display         flex
+    flex-grow       0
+    align-items     center
+    width           10em
+    cursor          pointer
+    font-weight     bold
+    text-transform  uppercase
 
-    .pho-viewer-toolbar-close
-        width           10em
-        font-weight     bold
-        text-transform  uppercase
-        display         flex
-        align-items     center
-        cursor          pointer
-        flex-grow       0
-
-        .pho-viewer-toolbar-close-text
-            font-size 14px
-
-        .pho-viewer-toolbar-close-cross
-            height   3em
-            width    3em
-            &::after, &::before
-                position         absolute
-                border-radius    3px
-                content          ''
-                height           2.5em
-                width            2px
-                background-color white
-                margin           .2em 1.4em
-            &::after
-                transform rotate(-45deg);
-            &::before
-                transform rotate(45deg);
+
+.pho-viewer-toolbar-close-text
+    font-size em(14px)
+
+
+.pho-viewer-toolbar-close-cross
+    width    3em
+    height   3em
+
+    &::after, &::before
+        position         absolute
+        margin           .2em 1.4em
+        border-radius    3px
+        width            2px
+        height           2.5em
+        background-color white
+        content          ''
+
+    &::after
+        transform rotate(-45deg);
+
+    &::before
+        transform rotate(45deg);
 
 
 @media (max-width: (1023/basefont)rem)
     .pho-viewer-toolbar
-        height              em(48px)
-        background-color    alpha(grey-08, .33)
         position            fixed
+        background-color    alpha(grey-08, .33)
+        height              em(48px)
```

Comment:
height before background, as background is inside the element, well that's the idea at least.